src/index.ts

```diff
@@ -178,3 +189,24 @@ function getPreloadLink(node: any) {
 
   return link;
 }
+
+function appendInlineTextIfNeeded(
+  pendingText: Text | null,
+  inlineTextHostNode: Node | null
+) {
+  if (pendingText && inlineTextHostNode) {
+    inlineTextHostNode.appendChild(pendingText);
+  }
+}
+
+function isInlineHost(node: Node) {
+  if (node.nodeType !== Node.ELEMENT_NODE) {
+    return false;
```

Comment:
this line is not being covered by tests, and by checking the [MDN nodeType docs](https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType) I think that the node here can't be anything else but an Element, so I am just going to remove this check, please let me know if you disagree